Expecto.netcore/Expecto.netcore.fsproj

```diff
@@ -0,0 +1,57 @@
+<Project ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props" />
+  <PropertyGroup>
+    <AssemblyName>Expecto.netcore</AssemblyName>
+    <Version>1.0.0-preview3</Version>
+    <OutputType>Library</OutputType>
+    <TargetFramework>netstandard1.6</TargetFramework>
+    <DebugType>pdbonly</DebugType>
+  </PropertyGroup>
+  <PropertyGroup Condition=" '$(Configuration)' == 'Release' ">
+    <DefineConstants>$(DefineConstants);RELEASE</DefineConstants>
+  </PropertyGroup>
+  <ItemGroup>
+    <Compile Include="..\paket-files\logary\logary\src\Logary.Facade\Facade.fs" />
+    <Compile Include="..\Expecto\Expecto.fs" />
+    <Compile Include="..\Expecto\Expect.fs" />
+    <EmbeddedResource Include="**\*.resx" />
+  </ItemGroup>
+  <ItemGroup>
+    <PackageReference Include="Microsoft.NET.Sdk">
+      <Version>1.0.0-alpha-20161104-2</Version>
+      <PrivateAssets>All</PrivateAssets>
+    </PackageReference>
+    <PackageReference Include="FSharp.Core">
+      <Version>4.0.1.7-alpha</Version>
+    </PackageReference>
+    <PackageReference Include="Argu">
+      <Version>3.3.0</Version>
+    </PackageReference>
+    <PackageReference Include="System.Linq.Parallel">
+      <Version>4.3.0</Version>
+    </PackageReference>
+
+
+    
+  </ItemGroup>
+  <ItemGroup Condition=" '$(TargetFramework)' == 'netstandard1.6' ">
```

Comment:
no need to split in another item group, there is only netstandard as target framework, merge with previous `ItemGroup`